src/pages/content/connectors/configuration.jsx

```diff
@@ -1,20 +1,360 @@
-import React from 'react'
+import React, { useState } from "react";
 
 import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiIcon,
+  EuiTitle,
+  EuiSpacer,
+  EuiCallOut,
+  EuiLink,
+  EuiText,
+  EuiButton,
+  EuiComboBox,
+  EuiFormRow,
+  EuiSelect,
+  useGeneratedHtmlId,
+  EuiFieldText,
+  EuiButtonEmpty
+} from "@elastic/eui";
 
-} from '@elastic/eui'
+import { CONNECTOR_ICONS } from '../../../components/icons/connector_icons';
```

Comment:
Can we do the alias import here? '@/components/...'

Reply:
Good point, for now I just was using in the same way (Actually I just copies and pasted this components from Kibana) It would be an improvement, but I guess following the same patterns like we do in Kibana would make us easy copy and paste code from there.
Component: https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/shared/icons/connector_icons.ts